templates/stub.yml

```diff
@@ -13,8 +15,20 @@ meta:
       backend_servers:
       - PLACEHOLDER-SERVER-IP
 
+  floatings:
+    ha_proxy:
+      public_z1: 172.16.2.6
+
```

Comment:
I think this should be a list of placeholder IPs, since not everyone will have just one IP, or use that network. Or maybe all the changes in stub.yml should be excluded, since it was built as a default that would get overridden for people as they use it?